Modules/Module_Processing/xASL_init_VisualizationSettings.m

```diff
@@ -141,12 +133,22 @@
     end
 
     ImageWB = xASL_io_Nifti2Im(x.D.Atlas.WholeBrain);
-    if x.S.bMasking(4)==0
+
+    if ~isfield(x.S,'bMasking') || isempty(x.S.bMasking)
+        bMasking = [1 1 1 1];
```

Comment:
Here they need defaulting to x.S as otherwise these parameters won't travel to the GetROIstats function. Alternatively, you remove it completely in this function (if we don't use x.S.bMasking elsewhere)?

It seems only necessary for the WBmask below, you can also just specify that one.

Reply:
That's the point. That we only use this locally and we don't set the initialization for the later functions here. We only need it for WBmask. See the reasonaning for that in my comment way above.

I added a comment explaining that bMasking is onyl used locally.